big-data/readme.md

```diff
@@ -1,3 +1,66 @@
+# Big Data Glossary  
+This project is a group project. The project includes a glossary of Big Data terminology. 
+Big Data is a term used to describe large amounts of data that can't be processed in the traditional way.  
+  
+Click on the letter to go to the term:  
+[A](A-D.md#a) &nbsp; [B](A-D.md#b) &nbsp; [C](A-D.md#c) &nbsp; [D](A-D.md#d) &nbsp; [E](E-M.md#e) &nbsp; [F](E-M.md#f) &nbsp; [G](E-M.md#g) &nbsp; [H](E-M.md#h) &nbsp; [I](E-M.md#i) &nbsp; [J](E-M.md#j) &nbsp; [K](E-M.md#k) &nbsp; [L](E-M.md#l) &nbsp; [M](E-M.md#m) &nbsp;  N  &nbsp; [O](O-S.md#o) &nbsp; [P](O-S.md#p) &nbsp; [Q](O-S.md#q) &nbsp; [R](O-S.md#r) &nbsp; [S](O-S.md#s) &nbsp; [T](T-Z.md#t) &nbsp; [U](T-Z.md#u) &nbsp; [V](T-Z.md#v) &nbsp; [W](T-Z.md#w) &nbsp; [X](T-Z.md/#x) &nbsp; Y &nbsp;  Z 
```

Comment:
This is not going to work for a few reasons
1. you're pointing to a file which you don't have (A-D.md doesn't exist in your directory and it shouldn't because your content is all inside the readme.
2. Instead you need to link to the sections below. For example [A](#a) &nbsp; [B](#b) &nbsp; and so on
3. IF there is no heading for the letter simply write the letter as follows: C &nbsp;

Reply:
Done, thank you!